src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -207,6 +207,37 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
     // of the extensions in the URLs. This allows loading assemblies with arbitrary filenames.
     assembliesBeingLoaded.forEach(r => addResourceAsAssembly(r, changeExtension(r.name, '.dll')));
     pdbsBeingLoaded.forEach(r => addResourceAsAssembly(r, r.name));
+
+    // Wire-up callbacks for satellite assemblies. Blazor will call these as part of the application
+    // startup sequence to load satellite assemblies for the application's culture.
+    window['Blazor']._internal.getSatelliteAssemblies = (dotnetArray: Pointer) : Pointer =>  {
```

Comment:
```suggestion
    window['Blazor']._internal.getSatelliteAssemblies = (cultures: System_Array<string>) : System_Object =>  {
```

... or something like that, if we can possibly be more explicit than `Pointer`.